### PR TITLE
feat: star on tile

### PR DESCRIPTION
**Pull request by taylorichigo**

### CODE REVIEW

File: components/starOnTile/starOnTile.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React,  { useState } from "react";
+import Rating from '@mui/material/Rating';
+
+const StarRating = (prop: {value: number}) => {  
+  return (
+       
+        <div className="flex pb-2">
+            <Rating
```
Comment (moncapitaine) [inline components/starOnTile/starOnTile.tsx:8]: can mouseover be deactivated..

File: components/receipeListTile/receipeListTile.tsx [hunk 2]
```diff
@@ -45,8 +47,20 @@ const ReceipeListTile: React.FC<RecipeListTileProps> = ({
         alt={recipe.name}
         src={recipe.image ?? ''}
       />
-      <h2 className="pt-1 text-xl font-extrabold text-center">{recipe.name}</h2>
-      <div className="flex gap-2 justify-center flex-wrap">
+      <div className="flex">
```
Comment (moncapitaine) [inline components/receipeListTile/receipeListTile.tsx:50]: grid?

File: components/starOnTile/starOnTile.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React,  { useState } from "react";
+import Rating from '@mui/material/Rating';
+
+const StarRating = (prop: {value: number}) => {
```
Comment (moncapitaine) [inline components/starOnTile/starOnTile.tsx:4]: add className

File: components/starOnTile/starOnTile.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React,  { useState } from "react";
+import Rating from '@mui/material/Rating';
+
+const StarRating = (prop: {value: number}) => {  
+  return (
+       
+        <div className="flex pb-2">
```
[the remark this answers is not in the document]
Comment (taylorichigo) [reply components/starOnTile/starOnTile.tsx:7]: Done

File: components/starOnTile/starOnTile.tsx (hunk 1 above)
Comment (taylorichigo) [reply components/starOnTile/starOnTile.tsx:8]: Added readOnly to value. Now its not changeable with mousover.

File: components/receipeListTile/receipeListTile.tsx (hunk 2 above)
Comment (taylorichigo) [reply components/receipeListTile/receipeListTile.tsx:50]: changed flex to grid layout

**Review by moncapitaine — APPROVED** [review]

**Review by moncapitaine — APPROVED** [review]

**Review by Bhoft — APPROVED** [review]

**Pull request merged**